src/xcode/ENA/ENA/Source/Developer Menu/Features/DMSettingsViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+#if !RELEASE
+
+import UIKit
+
+final class DMSettingsViewController: UITableViewController {
+	init(store: Store) {
+		self.store = store
+		super.init(style: .plain)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	private let store: Store
+
+	override func viewWillAppear(_ animated: Bool) {
+		navigationController?.setToolbarHidden(true, animated: animated)
+		super.viewWillAppear(animated)
+	}
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		tableView.rowHeight = UITableView.automaticDimension
+		tableView.register(DMOnOffCell.self, forCellReuseIdentifier: "DMOnOffCell")
+	}
+
+	// MARK: UITableView DataSource/Delegate
+	override func numberOfSections(in tableView: UITableView) -> Int {
+		1
+	}
+
+	override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		1
+	}
+
+	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		// swiftlint:disable:next force_cast
+		let cell = tableView.dequeueReusableCell(withIdentifier: "DMOnOffCell", for: indexPath) as! DMOnOffCell
+		
+		cell.configure(
+			with: .init(
+				title: "Fetch Hours instead of Days",
+				subtitle: "If enabled, only the last 3 hours are fetched.",
+				isOn: store.hourlyFetchingEnabled
+			)
+		)
+
+		cell.onOffDidChange = { isOn in
+			self.store.hourlyFetchingEnabled = isOn
+		}
+
+		return cell
+	}
+}
+
+private class DMOnOffCell: UITableViewCell {
```

Comment:
If I do that this class can no longer be private. At the moment the class is only used in this view controller and not tested (because it is developer related code). The same is also true for `DMMenuItem` btw. How do you handle this usually in "your own" projects?

Usually I try to split things up into multiple smaller frameworks/packages that allows me to basically have multiple access levels...

Reply:
that sounds reasonable to me. thanks for the info!